geometry_msgs/Pose2D poseA
geometry_msgs/Pose2D poseB
